Mathlib/Topology/Compactification/OnePoint.lean

```diff
@@ -72,6 +72,14 @@ scoped notation "∞" => OnePoint.infty
 /-- Coercion from `X` to `OnePoint X`. -/
 @[coe, match_pattern] def some : X → OnePoint X := Option.some
 
+protected lemma «forall» {p : OnePoint X → Prop} :
```

Comment:
The `@coe` tag in the previous def does not seem to fire, why cannot we simply write `x : OnePoint X` to mean `some x : OnePoint X`?

Reply:
It works after the `CoeTC`